### PR TITLE
delete childResource and cloudservices.properties.resources

### PR DESCRIPTION
**Pull request by grizzlytheodore**
delete childResource and cloudservices.properties.resources

### CODE REVIEW

File: generator/processors/Microsoft.Compute.ts [hunk 1]
```diff
@@ -60,9 +60,12 @@ export const postProcessor: SchemaPostProcessor = async (namespace, apiVersion,
     if (schema.resourceDefinitions?.cloudServices_updateDomains) {
         delete schema.resourceDefinitions.cloudServices_updateDomains;
     }
-    if (schema.resourceDefinitions?.cloudServices?.resources) {
+    if (schema.resourceDefinitions?.cloudServices?.properties?.resources) {
         delete schema.resourceDefinitions.cloudServices.resources
```
Comment (anthony-c-martin) [inline generator/processors/Microsoft.Compute.ts:64]: Missing `.properties` on this line

Comment (grizzlytheodore) [reply generator/processors/Microsoft.Compute.ts]: added. thank you

File: generator/processors/Microsoft.Compute.ts [hunk 2]
```diff
@@ -60,9 +60,12 @@ export const postProcessor: SchemaPostProcessor = async (namespace, apiVersion,
...
     }
+    if (schema.cloudServices_updateDomains_childResource) {
+        delete schema.cloudServices_updateDomains_childResource
+    }
```
Comment (anthony-c-martin) [inline generator/processors/Microsoft.Compute.ts:68]: I believe this should be `schema.definitions?.cloudServices_updateDomains_childResource`

Note that if you're using VSCode, you can find the JSON path of the cursor by looking at the breadcrumbs at the top of the editor:
image

**Review by anthony-c-martin — APPROVED** [review]

File: generator/processors/Microsoft.Compute.ts (hunk 2 above)
Comment (anthony-c-martin) [reply generator/processors/Microsoft.Compute.ts]: No worries. I've merged. I suggest just keeping an eye on the next generation pipeline run (should be ~24h from now) and making sure it has the intended effect

**Pull request merged**